### DISCUSSION TITLE
Welcome to ValidKube Discussions!

### DISCUSSION THREAD

**Discussion by Udi-Hofesh** [komodorio/validkube#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hasnainmakada-99** [comment]
Hello @Udi-Hofesh Will it be okay If I make non-code / Code contributions in validKube

**Comment by Udi-Hofesh** [comment]
Of course! feel free :)

*Udi Hofesh*
Developer Relations & Community
Tel: +972-58-7122985
Home: komodor.com <http://www.komodor.com>
Book a meeting with me <https://calendly.com/udi-hofesh/30min>
<https://komodor.com/blog/we-just-raised-our-series-b/?swcfpc=1>
